src/codegen/utilities/interpreter_helpers.py

```diff
@@ -34,12 +34,6 @@
     "SetSyncPulseTimeWhen",
 ]
 
-LIBRARY_INTERPRETER_IGNORED_FUNCTIONS = [
```

Comment:
Removing this as mentioned in this comment https://github.com/ni/nidaqmx-python/pull/267#discussion_r1165753231